assets/react/components/home.js

```diff
@@ -31,7 +35,9 @@ module.exports = React.createClass({
 		// }
 	},
 	componentDidMount: function(){
+		document.getElementsByClassName('cards-container')[0].style.visibility = "hidden";
 		this.initiateMasonry();
+		document.title = "Fodoo : Blogs for everyone";
```

Comment:
Same with this case. We can have a map for the titles of every page on frontend and add titles to props.